File: src/main/java/io/github/dqualizer/dqlang/types_new/dam/DomainArchitectureMapping.java
package io.github.dqualizer.dqlang.types_new.dam;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

import java.util.LinkedHashSet;

/**
 * Java class for the architecture-mapping
 */
@Getter
@ToString
public class DomainArchitectureMapping {

    private long version;
    private String context;
    @JsonProperty("server_info")
    private LinkedHashSet<ServerInfo> serverInfo;
    private LinkedHashSet<DomainArchitectureMappingObject> objects;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/DomainArchitectureMapping.kt
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping

/**
 * Java class for the architecture-mapping
 */
import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class DomainArchitectureMapping {
    val version: Long = 0
    val context: String? = null

    @JsonProperty("server_info")
    val serverInfo: LinkedHashSet<io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping.ServerInfo>? = null
    val objects: LinkedHashSet<io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping.DomainArchitectureMappingObject>? = null
}
